### ISSUE TITLE
"Cannot find module '../cli.js'" when following README setup for CLI

### ISSUE THREAD

**Issue by sebastian-onlea** [ckundo/auto-vo#19]
Having followed the configuration instructions (System Preferences, Voiceover Utility) and installed `auto-vo` with Node.js 16, I get an error trying to run the example command (`--url https://example.com`) for proof-of-life.

Same problem with Node.js 14...

Terminal session transcript:
```
sebastian@Sebastians-Onlea-Macbook-Pro / % sw_vers && npm -v && node -v                                     
ProductName:	macOS
ProductVersion:	12.1
BuildVersion:	21C52
8.3.2
v16.13.0
sebastian@Sebastians-Onlea-Macbook-Pro / % npm install -g auto-vo                                           

added 54 packages, and audited 55 packages in 947ms

4 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
sebastian@Sebastians-Onlea-Macbook-Pro / % npx auto-vo --url https://example.com --limit 5 --until 'Example'
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '../cli.js'
Require stack:
- /Users/sebastian/.nvm/versions/node/v16.13.0/lib/node_modules/auto-vo/bin/auto-vo.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/sebastian/.nvm/versions/node/v16.13.0/lib/node_modules/auto-vo/bin/auto-vo.js:3:17)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/sebastian/.nvm/versions/node/v16.13.0/lib/node_modules/auto-vo/bin/auto-vo.js'
  ]
}
```

**Comment by sebastian-onlea** [comment]
Trying to eliminate variables - I tried it using the system-global Node.js environment (outside `nvm`), and got the same issue.

```
sebastian@Sebastians-Onlea-Macbook-Pro ~ % sw_vers && npm -v && node -v 
ProductName:    macOS
ProductVersion: 12.1
BuildVersion:   21C52
8.3.2
v14.18.1
sebastian@Sebastians-Onlea-Macbook-Pro ~ % sudo npm install -g auto-vo                                      

added 54 packages, and audited 55 packages in 3s

4 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
sebastian@Sebastians-Onlea-Macbook-Pro ~ % npx auto-vo --url https://example.com --limit 5 --until 'Example'
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module '../cli.js'
Require stack:
- /usr/local/lib/node_modules/auto-vo/bin/auto-vo.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/auto-vo/bin/auto-vo.js:3:17)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/local/lib/node_modules/auto-vo/bin/auto-vo.js' ]
}
```

**Comment by jklingen** [comment]
I had the same problem, some additional observations:

- the file `cli.js` actually is not present in the `auto-vo` directory or its subdirectories
- when executing the same command with `auto-vo@1.4.0` it works as expected

**Comment by ckundo** [comment]
thanks y'all, this is resolved in `v1.6.0`
